setup.py

```diff
@@ -1,1137 +1,9 @@
-        long_description_content_type='text/markdown',
-        ext_modules=ext_modules,
-        cmdclass=cmdclass,
-        packages=find_packages(include=['kivy*']),
-        package_dir={'kivy': 'kivy'},
-        package_data={
-            'kivy':
-                glob_paths('*.pxd', '*.pxi') +
-                glob_paths('**/*.pxd', '**/*.pxi') +
-                glob_paths('data/**/*.*') +
-                glob_paths('include/**/*.*') +
-                glob_paths('tools/**/*.*', excludes=('.pyc', '.enc')) +
-                glob_paths('graphics/**/*.h') +
-                glob_paths('tests/**/*.*') +
-                [
-                    'setupconfig.py',
-                ] + binary_deps
-        },
-        data_files=[] if split_examples else list(examples.items()),
-        classifiers=[
-            'Development Status :: 5 - Production/Stable',
-            'Environment :: MacOS X',
-            'Environment :: Win32 (MS Windows)',
-            'Environment :: X11 Applications',
-            'Intended Audience :: Developers',
-            'Intended Audience :: End Users/Desktop',
-            'Intended Audience :: Information Technology',
-            'Intended Audience :: Science/Research',
-            'License :: OSI Approved :: MIT License',
-            'Natural Language :: English',
-            'Operating System :: MacOS :: MacOS X',
-            'Operating System :: Microsoft :: Windows',
-            'Operating System :: POSIX :: BSD :: FreeBSD',
-            'Operating System :: POSIX :: Linux',
-            'Programming Language :: Python :: 3.5',
-            'Programming Language :: Python :: 3.6',
-            'Programming Language :: Python :: 3.7',
-            'Topic :: Artistic Software',
-            'Topic :: Games/Entertainment',
-            'Topic :: Multimedia :: Graphics :: 3D Rendering',
-            'Topic :: Multimedia :: Graphics :: Capture :: Digital Camera',
-            'Topic :: Multimedia :: Graphics :: Presentation',
-            'Topic :: Multimedia :: Graphics :: Viewers',
-            'Topic :: Multimedia :: Sound/Audio :: Players :: MP3',
-            'Topic :: Multimedia :: Video :: Display',
-            'Topic :: Scientific/Engineering :: Human Machine Interfaces',
-            'Topic :: Scientific/Engineering :: Visualization',
-            ('Topic :: Software Development :: Libraries :: '
-             'Application Frameworks'),
-            'Topic :: Software Development :: User Interfaces'])
-else:
-    setup(
-        name='Kivy-examples',
-        version=__version__,
-        author='Kivy Team and other contributors',
-        author_email='kivy-dev@googlegroups.com',
-        url='http://kivy.org',
-        license='MIT',
-        description=('Kivy examples.'),
-        long_description_content_type='text/markdown',
-        long_description=get_description(),
-        data_files=list(examples.items()))
+import subprocess
+subprocess.call(['python', 'msetup.py'] + sys.argv[1:])
```

Comment:
no that we can't do, at the very least we should use sys.executable to make sure we run it with the same python version that called us, but i think just importing the module and calling a function from it would be better.